react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -143,6 +144,7 @@ const SelectorItem: FC<Props> = ({
         <div
           className={classNames(handles.valueWrapper, {
             [`${handles.skuSelectorItemTextValue} c-on-base center pl5 pr5 z-1 t-body`]: !isImage,
+            [handles.unAvailable]: !isAvailable,
```

Comment:
Here you are creating a new CSS handle, instead of a modifier to an existing one.

To create a modifier, you should just use the `applyModifiers` function in a way that's almost identical to how we do it in line 75 :)

You probably want to add this `unavailable` modifier to `handles.valueWrapper`. Using `applyModifiers` will make sure that the div gets both `valueWrapper` and `valueWrapper--unavailable`, when the product is indeed unavailable.